src/main/java/io/jenkins/plugins/pipelinegraphview/utils/PipelineNodeUtil.java

```diff
@@ -47,11 +50,43 @@ public static String getDisplayName(@NonNull FlowNode node) {
                 : name;
     }
 
+    public static boolean isStep(FlowNode node) {
+        if (node != null) {
+            if (node instanceof AtomNode) {
+                return true;
+            }
+            if (node instanceof StepStartNode) {
+                StepStartNode stepStartNode = (StepStartNode) node;
+                boolean takesImplicitBlockArgument = false;
+                StepDescriptor sd = stepStartNode.getDescriptor();
+                if (sd != null) {
+                    takesImplicitBlockArgument = sd.takesImplicitBlockArgument();
+                }
+                return !isStage(node)
+                        && !isParallelBranch(node)
+                        && stepStartNode.isBody()
+                        && !takesImplicitBlockArgument;
```

Comment:
There steps that generate `StepStartNode` nodes in the pipeline graph/tree that we want to visualize as steps. Unfortunately identifying those steps among other nodes (`stage`, `parallel`, and other steps that take an implicit block argument) is awkward. I haven't been able to find a more elegant solution after searching through the relevant APIs.